### ISSUE TITLE
Download pages post 1.0

### ISSUE THREAD

**Issue by vim-usds** [usds/justice40-tool#2088]

**Comment by vim-usds** [comment]
![Screen Shot 2022-11-17 at 11 32 05 AM](https://user-images.githubusercontent.com/86254807/202556062-eafc8449-ee2b-4973-bf53-9972edfd9d3f.png)

**Comment by KameronKerger** [comment]
no longer waiting for CEQ on the TSD
file size done
need to zip the xlsx and csv with the codebook

**Comment by KameronKerger** [comment]
need to add the release notes into the zip files release-notes.txt

**Comment by vim-usds** [comment]
@KameronKerger - will spend some time figuring out if things are missing from code book.

@mattbowen-usds - once this is figured out, let's schedule a time to rrun the previous versions GHA to update S3 to the latest downloadable files and update the local file systems as well

**Comment by KameronKerger** [comment]
the shapefile zip is all messed up. there are now two codebooks. it is very confusing and people are confused. especially about which column is identified as disadvantaged which is a pretty important one. I believe it is SN_C but that is only referenced in one codebook (not the 1.0 codebook). can someone help out on this?
<img width="751" alt="Screen Shot 2022-12-19 at 12 23 50 PM" src="https://user-images.githubusercontent.com/77702996/208514632-2980b8db-22e7-4ef3-a033-63353e3c9504.png">

**Comment by vim-usds** [comment]
@KameronKerger 

It seems the Shapefile on the UI:

![Screen Shot 2022-12-19 at 1.08.59 PM.png](https://images.zenhubusercontent.com/60d4d92c1bf5e498d6e808ba/1257197d-da12-48d7-b33f-f1fe912a0f71)

Points to the following _zipped_ file: `1.0-shapefile-codebook`. Unzipping this file has 3 files in it:

1. readme-version-1.0.md
2. 1.0-codebook.csv
3. usa.zip

The first file `usa.zip` has the following:

1. columns.csv
2. usa.cpg
3. usa.shx
4. usa.prj
5. usa.dbf
6. usa.shp

### Questions

1. Should `1.0-shapefile-codebook` have the 3 aforementioned files?
2. It is safe to say that usa.zip (which seems to be the shapefile) is somehow not correct? How were you able to tell?

I'm asking these to figure out what could be wrong and how we can fix it. Thanks!

**Comment by vim-usds** [comment]
@KameronKerger - I think I might be following. Is it that `columns.csv` is codebook and so is `1.0-codebook.csv`?

**Comment by KameronKerger** [comment]
yes indeed @vim-usds. there are now two codebooks. 1.0 looks like Emma made for 1.0 but it seems to lack the column names that are in columns.csv? I think both are actually required unless we can add the column names to the 1.0 codebook? and the readme file should be a txt file. and ideally the folder is not called just usa.zip.

**Comment by vim-usds** [comment]
Thanks @KameronKerger. I plan to discuss this with @mattbowen-usds . I think he’s out sick today and so hopefully he’s back tomorrow.

Also, If we know what needs to change, we can manually do it and push the changes up to S3 to get the fix in today.

**Comment by KameronKerger** [comment]
maybe just loop me in when you are talking to Matt?

**Comment by vim-usds** [comment]
@KameronKerger - will do

**Comment by vim-usds** [comment]
There are 3 issues that this ticket aims to resolve:

1. `usa.zip` should be renamed to `cejst_version1_communites`
2. There are currently two codebooks. we would like to combine the abbreviations for example`SN_C` to the full column names
3. Update the README
